### ISSUE TITLE
Can't get it to render or play files from Korean PC version of Digimon World

### ISSUE THREAD

**Issue by lampuiho** [frabert/libdmusic#20]
[BGM.zip](https://github.com/libdmusic/libdmusic/files/4349338/BGM.zip)
When using render, most get stuck. For 019, it outputs a file but no sound.
EDIT: It's the same issue with the lack of style sheet.

**Comment by frabert** [comment]
Hi @lampuiho! I've inspected the files and it seems you are in a similar situation as #18 

In essence, I'm planning on rewriting the library from scratch sometime in the future, as this is one of the cases it currently doesn't handle very well. You're probably better off writing a custom parser for your kind of files, and then using a library like fluidsynth to render the music with the DLS files.

**Comment by lampuiho** [comment]
Yep, I will see if MaKiPL's code, which seems to do what you suggested, works first and see where it gets stuck. But that will have to come after I finish my thesis and graduate XD
